client/src/api/publisher.ts

```diff
@@ -30,3 +55,22 @@ export async function getPublishers(body: PaginationRequest) {
   //w takim przypadku można by coś zlogować i wyświetlić stosowny komunikat
   return PublisherSearchResponse.parse(data);
 }
+
+export async function getPublisher(id: string) {
+  //endpoint powinien działać zarówno dla zalogowanego i anonima
+  let auth = '';
```

Comment:
to się powtarza, warto wydzielić czy nie ma co?

Reply:
ale co dokładnie się powtarza? to dla zalogowanego lub nie?

Reply:
pobieranie authTokena w bloku try/catch